build/media_source/templates/site/cassiopeia/scss/system/searchtools/searchtools.scss

```
@@ -48,3 +48,9 @@
     margin-inline-start: 8px;
   }
 }
+
+.js-stools-field-selector {
+  .form-select {
+    width:auto;
```

```suggestion
    width: auto;
```
Fix CS error in Drone